inst/tinytest/test-app-content-handlers.R

```diff
@@ -63,4 +63,21 @@ err_msg = paste0("lexical error: invalid char in json text.\n",
 expect_equal(rs[[1]], err_msg)
 expect_equal(rs[[2]], "text/plain")
 
+
+# Test static files
+files <- c("test.R", "test.txt", "testdata.csv", "testdata.rds", "testplot.jpg",
+           "testplot.pdf", "testplot.png")
+request_static_files = Request$new(path = "/list_static_files")
+rs = backend$convert_response(app$process_request(request_static_files))
+expect_equal(jsonlite::fromJSON(rs[[1]]), files)
```

Comment:
order of files in the `files` array and when we list files in the dir might be different. So we need to compare sets / sorted lists.

Reply:
This is handled by the changes in da1d045